### PR TITLE
Sprinkle some UI polish for description section

### PR DESCRIPTION
**Pull request by donokuda**
Couple of small tweaks to the PR description:

- Shrink avatar to 20px
- Minor margin/padding adjustments
- Shrink images if they are included in the markdown file.

body-changes

/cc @grokys since this targets your branch

### CODE REVIEW

File: src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml [hunk 1]
```diff
@@ -41,6 +41,14 @@
                 <Setter Property="FontSize" Value="12"/>
                 <Setter Property="TextAlignment" Value="Left"/>
                 <Setter Property="PagePadding" Value="0"/>
+
+                <Style.Resources>
+                    <Style TargetType="Image">
+                        <Setter Property="Stretch" Value="UniformToFill" />
+                        <Setter Property="StretchDirection" Value="DownOnly" />
+                        <Setter Property="MaxWidth" Value="{Binding ActualWidth, ElementName=bodyMarkdown}" />
```
Comment (donokuda) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml]: Is there a better way to bind this value? Coupling it specifically to `ElementName` doesn't seem very smart or reusable. 🔬 🐶

File: src/GitHub.App/SampleData/PullRequestDetailViewModelDesigner.cs [hunk 2]
```diff
@@ -23,7 +23,9 @@ public PullRequestDetailViewModelDesigner()
 
 ViewModels are responsible for handling the UI on the view they control, but they shouldn't be handling UI for things outside of the view. In this case, we're showing errors in VS outside the view, and that should be handled by the section that is hosting the view.
 
-This requires that errors be propagated from the viewmodel to the view and from there to the host via the IView interface, since hosts don't usually know what they're hosting.";
+This requires that errors be propagated from the viewmodel to the view and from there to the host via the IView interface, since hosts don't usually know what they're hosting.
+
+![An image](https://cloud.githubusercontent.com/assets/1174461/18882991/5dd35648-8496-11e6-8735-82c3a182e8b4.png)";
```
Comment (donokuda) [inline src/GitHub.App/SampleData/PullRequestDetailViewModelDesigner.cs]: Are there better ways to test images rendering?

File: src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml (hunk 1 above)
Comment (grokys) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml]: I suspect here that using a `RelativeSource AncestorType` binding might be the best way to go: http://stackoverflow.com/questions/84278/how-do-i-use-wpf-bindings-with-relativesource

File: src/GitHub.App/SampleData/PullRequestDetailViewModelDesigner.cs (hunk 2 above)
Comment (grokys) [inline src/GitHub.App/SampleData/PullRequestDetailViewModelDesigner.cs]: Not sure what you mean... Do you mean instead of using a URL?

**Comment by grokys** [comment]
Merging and will tidy up the markup relating to the 2 questions myself (if it works ;) )

File: src/GitHub.App/SampleData/PullRequestDetailViewModelDesigner.cs (hunk 2 above)
Comment (grokys) [inline src/GitHub.App/SampleData/PullRequestDetailViewModelDesigner.cs]: Actually if that is what you mean, I think a URL is best. We could probably reference a file in our resources, but that would mean that the file was distributed with the extension unnecessarily.

**Pull request merged**